1.22/Dockerfile.rhel8

```diff
@@ -45,10 +45,10 @@ RUN yum -y module enable nginx:$NGINX_VERSION && \
     yum -y clean all --enablerepo='*'
 
 # Copy the S2I scripts from the specific language image to $STI_SCRIPTS_PATH
-COPY ./s2i/bin/ $STI_SCRIPTS_PATH
+COPY 1.22/s2i/bin/ $STI_SCRIPTS_PATH
```

Comment:
Why somewhere is used `$NGINX_VERSION` and somewhere a hardcoded version in these cases?

Reply:
RHEL-8 somehow does not like it. I'm not sure whether it's a weird version of podman or what, but just simple change like this https://github.com/sclorg/nginx-container/pull/184/commits/7af4049fb857cccb674d00bda444795ddaf9c07a fixed the CI. This is still a bit mistery to me though.